isis/src/qisis/objs/Footprint2DView/Footprint2DView.h

```diff
@@ -165,7 +171,7 @@ namespace Isis {
       QToolBar *m_activeToolBar; //!< The active tool bar
       ToolPad *m_toolPad; //!< The tool pad
 
-      QAction *m_controlNetTool;  //!< The Control Point Editor Tool
+      QAction *m_controlNetToolAction;  //!< The action for Control Point Editor Tool(MosaicControl
```

Comment:
I forgot to remove this with my last PR. This is not used anymore, so we can remove it.